flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/RestHandlerConfiguration.java

```diff
@@ -78,10 +83,17 @@ public static RestHandlerConfiguration fromConfiguration(Configuration configura
 		final String rootDir = "flink-web-ui";
 		final File webUiDir = new File(configuration.getString(WebOptions.TMP_DIR), rootDir);
 
+		final long metricFetcherUpdateInterval = configuration.getLong(WebOptions.METRIC_FETCHER_UPDATE_INTERVAL);
+
 		return new RestHandlerConfiguration(
 			refreshInterval,
 			maxCheckpointStatisticCacheEntries,
 			timeout,
-			webUiDir);
+			webUiDir,
+			metricFetcherUpdateInterval);
+	}
+
+	public long getMetricFetcherUpdateInterval() {
+		return 0L;
```

Comment:
this must return `metricFetcherUpdateInterval`.

Reply:
Removed this method altogether.